includes/WebRequest.php

```diff
@@ -171,7 +171,9 @@ public static function detectServer() {
 			}
 			$host = $parts[0];
 			if ( $parts[1] === false ) {
-				if ( isset( $_SERVER['SERVER_PORT'] ) ) {
+				if ( !empty( $_SERVER['HTTP_FASTLY_SSL'] ) ) {
+					$port = 443;
+				} elseif ( isset( $_SERVER['SERVER_PORT'] ) ) {
 					$port = $_SERVER['SERVER_PORT'];
 				} // else leave it as $stdPort
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app